### ISSUE TITLE
RCS1057 False Negative

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#485]
RCS1057 isn't flagged for any of these properties. Is it a false negative?

```csharp
namespace RCS1057_False_Negative
{
    static class Program
    {
        public static string A { get; set; }
        public static string B { get; set; }
        public static string C { get; set; }
        public static string D { get; set; }
    }
}
```

**Comment by JosefPihrt** [comment]
This analyzer does not enforce an empty line between two single-line declaration. I'll update documentation accordingly.
